### DISCUSSION TITLE
Adding new font to Imagemagic on windows

### DISCUSSION THREAD

**Discussion by mchdry** [ImageMagick/ImageMagick#7743] [Help]
### ImageMagick version

7.1.1 Q16

### Operating system, version and so on

Windows 11

### Description

Hi all

Trying to add fonts onto imagemagick, so they load up on Mogrify plugin for Lightroom. Trying to install Inter font from Google Fonts. I have installed the Inter.ttc on windows, it shows up everywhere, however it does not show up on imagemagick.

When I do magick -list font, it does not show up there. 

I have tried adding it to type-ghostscript.xml and type.xml using the below script, however it doesnt work. What am I doing wrong?

`<type name="Inter" fullname="Inter" family="Inter" foundry="" weight="400" style="normal" stretch="normal" format="ttc" metrics="" glyphs="C:\Windows\Fonts\Inter.ttc"/>`

**Comment by mchdry** [comment]
Gave up and just used Arial font.

**Comment by dlemstra** [comment]
When you install a font it will be installed for the current user. What happens when you install it for all users?

**Reply by mchdry** [reply]
The font is installed for all users :)

**Reply by dlemstra** [reply]
Wonder if we even have support for ttc fonts on Windows. Will need to check that.

**Reply by mchdry** [reply]
I assume it would?

When I run the magick -list font command, it shows ttc files as well.

**Comment by snibgo** [comment]
Where did you get inter.ttc? I downloaded inter.zip from https://fonts.google.com/specimen/Inter , but that zip contains only ttf files.

On Windows 11, IM seems to handle ttc files:
```
magick -list font |grep -i ttc
    glyphs: c:\windows\fonts\cambria.ttc
    glyphs: c:\windows\fonts\msjh.ttc
    glyphs: c:\windows\fonts\msjhbd.ttc
    glyphs: c:\windows\fonts\msjhl.ttc
    glyphs: c:\windows\fonts\msyh.ttc
    glyphs: c:\windows\fonts\msyhbd.ttc
    glyphs: c:\windows\fonts\msyhl.ttc
    glyphs: c:\windows\fonts\mingliub.ttc
    glyphs: c:\windows\fonts\msgothic.ttc
    glyphs: c:\windows\fonts\simsun.ttc
    glyphs: c:\windows\fonts\yugothb.ttc
    glyphs: c:\windows\fonts\yugothl.ttc
    glyphs: c:\windows\fonts\yugothm.ttc
    glyphs: c:\windows\fonts\yugothr.ttc
```

**Reply by mchdry** [reply]
From here https://rsms.me/inter/

Ill download the set from google and see if that works :)

**Reply by mchdry** [reply]
Just confirming, I do have to add that string into type.xml or type-ghostscript, yes?

**Reply by dlemstra** [reply]
On Windows the registry of the machine is used. This might require a reboot.

**Reply by mchdry** [reply]
Did a restart, still not showing up.
